def.cf

```diff
@@ -266,13 +266,20 @@ bundle common inventory_control
 # This common bundle is for controlling whether some inventory bundles
 # are disabled.
 {
-  vars:
+  defaults:
+
       "dmidecoder" string => "/usr/sbin/dmidecode";
       "lldpctl_exec" string => "/usr/bin/lldpctl";
       "lsb_exec" string => "/usr/bin/lsb_release";
       "mtab" string => "/etc/mtab";
       "proc" string => "/proc";
 
+  vars:
+
+    freebsd::
+
+      "dmidecoder" string => "/usr/local/sbin/dmidecode";
```

Comment:
Better IMO, because `defaults` can have issues:

```
"dmidecoder" string => ifelse("freebsd", "/usr/local/sbin/dmidecode", "/usr/sbin/dmidecode");
```